modular_skyrat/modules/implants/code/augments_arms.dm

```diff
@@ -36,6 +52,95 @@
 	items_list += WEAKREF(new /obj/item/melee/implantarmblade/energy(src))
 	return TRUE
 
+/obj/item/knife/razor_claws
+	name = "implanted razor claws"
+	desc = "A set of sharp, retractable claws built into the fingertips, five double-edged blades sure to turn people into mincemeat. Capable of shifting into 'Precision' mode to act similar to wirecutters."
+	icon = 'modular_skyrat/modules/implants/icons/razorclaws.dmi'
+	righthand_file = 'modular_skyrat/modules/implants/icons/razorclaws_righthand.dmi'
+	lefthand_file = 'modular_skyrat/modules/implants/icons/razorclaws_lefthand.dmi'
+	icon_state = "wolverine"
+	inhand_icon_state = "wolverine"
+	var/knife_force = 10
+	w_class = WEIGHT_CLASS_BULKY
+	var/knife_wound_bonus = 5
+	var/cutter_force = CUTTER_FORCE
+	var/cutter_wound_bonus = CUTTER_WOUND_BONUS
+	var/cutter_bare_wound_bonus = CUTTER_WOUND_BONUS
+	tool_behaviour = TOOL_KNIFE
+	toolspeed = 1
+	item_flags = NEEDS_PERMIT //Beepers gets angry if you get caught with this.
+
+/obj/item/knife/razor_claws/attack_self(mob/user)
+	playsound(get_turf(user), 'sound/items/change_drill.ogg', 50, TRUE)
+	if(tool_behaviour != TOOL_WIRECUTTER)
+		tool_behaviour = TOOL_WIRECUTTER
+		to_chat(user, span_notice("You shift [src] into Precision mode, for wirecutting."))
+		icon_state = "precision_wolverine"
+		inhand_icon_state = "precision_wolverine"
+		flags_1 = CONDUCT_1
+		force = cutter_force
+		wound_bonus = cutter_wound_bonus
+		bare_wound_bonus = cutter_bare_wound_bonus
+		sharpness = NONE
+		hitsound = CUTTER_HITSOUND
+		usesound = CUTTER_USESOUND
+		attack_verb_continuous = CUTTER_ATTACK_VERB_CONTINUOUS
+		attack_verb_simple = CUTTER_ATTACK_VERB_SIMPLE
+
+	else
+		tool_behaviour = TOOL_KNIFE
+		to_chat(user, span_notice("You shift [src] into Killing mode, for slicing."))
+		icon_state = "wolverine"
+		inhand_icon_state = "wolverine"
+		force = knife_force
+		sharpness = KNIFE_SHARPNESS
+		wound_bonus = knife_wound_bonus
+		bare_wound_bonus = KNIFE_BARE_WOUND_BONUS
+		hitsound = KNIFE_HITSOUND
+		usesound = KNIFE_USESOUND
+		attack_verb_continuous = KNIFE_ATTACK_VERB_CONTINUOUS
+		attack_verb_simple = KNIFE_ATTACK_VERB_SIMPLE
+
+/obj/item/knife/razor_claws/attackby(obj/item/stone, mob/user, param)
+	if(!istype(stone, /obj/item/scratching_stone))
+		return ..()
+
+	knife_force = ENHANCED_KNIFE_FORCE
+	knife_wound_bonus = ENHANCED_KNIFE_WOUND_BONUS
+	armour_penetration = ENHANCED_KNIFE_ARMOR_PENETRATION //Let's give them some AP for the trouble.
```

Comment:
This should check if the claws are currently in knife mode to set ``force = knife_force`` and ``wound_bonus = knife_wound_bonus`` after they've been set.